### ISSUE TITLE
Creating "ODN" archives

### ISSUE THREAD

**Issue by namejustforthis** [morkt/GARbro#642]
Would it be possible to add the ability to create ".odn" archives, the format that Valkyria/Odin-Soft games use?

**Comment by HOKORISAMA** [comment]
It's possible but the game engine only supports full-width characters.
Here is the Modified  [Garbro](https://github.com/satan53x/GARbro) that can create odn archives.

**Comment by namejustforthis** [comment]
Thank so much for the reply. I assume you would have mentioned it if there is; but is there no known way to modifiy the games to change that?

**Comment by HOKORISAMA** [comment]
> Thank so much for the reply. I assume you would have mentioned it if there is; but is there no known way to modifiy the games to change that?

It's not like there is no solution, there are many solutions to this like you can modify the game exe but for that you'll need to know how win32 and asm works, you can try writing dll hook to hook the text rendering function and change the parameters for that functions.


The easiest method is you can try frida-tools that can work in python and JS, write a basic hook first to retrieve the address of the function you want to hook. You can use xdbg64 to debug the exe file then check the module name and try to determine which module is responsible for text rendering/Font rendering like CreateFontA/CreateFontIndirectA and so on.

In short it's a really lengthy process with little hope to work.
